src/main/java/roomescape/time/dto/ReservationTimeRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package roomescape.time.dto;
+
+import jakarta.validation.constraints.NotBlank;
+
+import roomescape.time.domain.ReservationTime;
+
+public record ReservationTimeRequestDto(
+        @NotBlank(message = "예약 시간을 입력해야 합니다.") String startAt) {
```

Comment:
```suggestion
        @NotBlank(message = "예약 시간을 입력해야 합니다.") String startAt
) {
```

ReservationTimeResponseDto 랑 컨벤션이 달라서 둘중 하나로 통일되면 좋을것 같아요